spec/controllers/super_admin/app_config_controller_spec.rb

```diff
@@ -34,13 +34,13 @@
     context 'when it is an aunthenticated super admin' do
```

Comment:
Should we include an additional specification to verify that the allowed_config does not contain customization variables intended for the 'enterprise' plan?"

Reply:
Not too worried about this at the moment since the config reset job which we are planning will reset it back to the old values.  also, there is always the option to update directly from the database/console so 🤷‍♂️

Reply:
😅